sdk/evaluation/azure-ai-evaluation/tests/e2etests/test_evaluate.py

```diff
@@ -114,18 +114,18 @@ def test_evaluate_with_groundedness_evaluator(self, model_config, data_file):
         assert result["rows"] is not None
         assert row_result_df.shape[0] == len(input_data)
 
-        assert "outputs.grounded.gpt_groundedness" in row_result_df.columns.to_list()
+        assert "outputs.grounded.groundedness" in row_result_df.columns.to_list()
```

Comment:
Is there a supportive change being made on UI to render this to the correct column?

Reply:
Yes